README.md

```diff
@@ -65,9 +65,12 @@ Each requirement is configured with the options provided by that Chart. Please c
 | `pgadmin.enabled`                                                           | Enable PgAdmin                                                                                                     | `true`                          |
 | `openldap.enabled`                                                          | Enable openldap                                                                                                    | `true`                          |
 | `phpldapadmin.enabled`                                                      | Enable phpLDAPadmin                                                                                                | `true`                          |
-| `elasticstack.enabled`                                                      | Enable Elastic-stack                                                                                               | `false`                         |
 | `kafka.enabled`                                                             | Enable Kafka                                                                                                       | `false`                         |
 | `cassandra.enabled`                                                         | Enable Cassandra                                                                                                   | `false`                         |
+| `filebeat.enabled`                                                         | Enable Filebeat                                                                                                   | `false`                         |
```

Comment:
@AyadiAmen what is the purpose of filebeat? As ELK = Elasticksearch, Logstash, Kibana

Reply:
@alexnuttinck Filebeat is used to easily ship log file data to Logstash and Elasticsearch to centralize your logs and analyze them in real time, Logstash is useful if you need to  apply some common transformations and filtering to your events.

Reply:
@AyadiAmen ok, thanks for the explanation.